aspnetcore/fundamentals/middleware.md

```diff
@@ -188,17 +188,17 @@ app.Map("/level1/level2", HandleMultiSeg);
 
 ## Built-in middleware
 
-ASP.NET Core ships with the following middleware components:
-
-| Middleware | Description |
-| ----- | ------- |
-| [Authentication](xref:security/authentication/identity) | Provides authentication support. |
-| [CORS](xref:security/cors) | Configures Cross-Origin Resource Sharing. |
-| [Response Caching](xref:performance/caching/middleware) | Provides support for caching responses. |
-| [Response Compression](xref:performance/response-compression) | Provides support for compressing responses. |
-| [Routing](xref:fundamentals/routing) | Defines and constrains request routes. |
-| [Session](xref:fundamentals/app-state) | Provides support for managing user sessions. |
-| [Static Files](xref:fundamentals/static-files) | Provides support for serving static files and directory browsing. |
+ASP.NET Core ships with the following middleware components, as well as a description of the order in which they run:
+
+| Middleware | Description | Ordering |
+| ----- | ------- | ------- |
+| [Authentication](xref:security/authentication/identity) | Provides authentication support. | Before `HttpContext.User` is needed. Terminal for OAuth callbacks. |
+| [CORS](xref:security/cors) | Configures Cross-Origin Resource Sharing. | Before components that use CORS |
+| [Response Caching](xref:performance/caching/middleware) | Provides support for caching responses. | Before components that want caching |
+| [Response Compression](xref:performance/response-compression) | Provides support for compressing responses. | Before components that want to use compression |
+| [Routing](xref:fundamentals/routing) | Defines and constrains request routes. |  |
+| [Session](xref:fundamentals/app-state) | Provides support for managing user sessions. |  |
+| [Static Files](xref:fundamentals/static-files) | Provides support for serving static files and directory browsing. | Terminal if a request matches files |
 | [URL Rewriting Middleware](xref:fundamentals/url-rewriting) | Provides support for rewriting URLs and redirecting requests. |
```

Comment:
Before components that consume the URL.

Why does this one include `Middleware` when the others don't?

Reply:
"Why does this one include Middleware when the others don't?" 
My guess: The author was probably moonlighting at the Department of Redundancy Department at the time. I can remove it.

Reply:
We had a reason for doing it :point_right: https://github.com/aspnet/Docs/pull/2494 (January 11, 2017)

A year later tho, I don't recall why. I think Tom and I spoke about it off the PR ... @tdykstra do you recall why we did that?

Also note that it's that way in the TOC, so u could change it back there, too.